scripts/cem-plugin-react-wrapper.js

```diff
@@ -175,8 +175,10 @@ export default function genReactWrapper({
             const fileImports = modules
                 .filter(
                     (m) =>
-                        m.declarations.length === 0 &&
-                        m.path.indexOf('sync/') === -1
+                        m.exports.length === 1 &&
+                        m.exports.some(
```

Comment:
I changed this line to Array.some(...), but I have to keep L178 as it is now because Tooltip component has a special customer element declaration - TooltipProxy in `src/Tooltip.ts`.